dotnet/src/SemanticKernel.Abstractions/IKernel.cs

```diff
@@ -190,6 +167,19 @@ public interface IKernel
 
     #region Obsolete
 
+    /// <summary>
+    /// Semantic memory instance
+    /// </summary>
+    [Obsolete("Memory functionality will be placed in separate Microsoft.SemanticKernel.Plugins.Memory package. This will be removed in a future release.")]
```

Comment:
We should include some information so a developer know what to do. Can we include a link to a kernel syntax example here?